types/bitcoin-computer/index.d.ts

```diff
@@ -0,0 +1,42 @@
+// Type definitions for @bitcoin-computer/lib 0.9
+// Project: https://github.com/bitcoin-computer/monorepo (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
```

Comment:
```suggestion
// Project: https://github.com/bitcoin-computer/monorepo
```

Reply:
Yes, this change would be an improvement